sdk/containerregistry/azure-containerregistry/tests/testcase.py

```diff
@@ -234,6 +226,7 @@ def import_image(repository, tags):
 
 @pytest.fixture(scope="session")
 def load_registry():
+    return
```

Comment:
is this meant to be here?

Reply:
Nope, I put that there when running locally and sometimes accidentally commit it.